assets/pages/container/[id].vue

```diff
@@ -16,11 +16,8 @@ const { id } = defineProps<{ id: string }>();
 const currentContainer = store.currentContainer($$(id));
 const { activeContainers, ready } = storeToRefs(store);
 
-setTitle("loading");
-
-onMounted(() => {
-  setTitle(currentContainer.value?.name);
+watchEffect(() => {
+  if (ready.value === true)
+    currentContainer.value !== undefined ? setTitle(currentContainer.value.name) : setTitle("Not Found");
```

Comment:
Maybe here it could just set loading if not ready yet?

Reply:
The reason I removed the loading title, is because it is loaded almost instantly.
It only needs to load the most basic container information, and not the logs.
Also, if the title would be set to loading first, it would almost instantly be set to the container name, which doesn't look very good IMO

Reply:
> The reason I removed the loading title, is because it is loaded almost instantly.

That might be true for you. :) And I have put a lot of effort to make sure that it is fast. However, there is probably about 2% of Dozzle users who have multiple hosts with 100s of containers. That probably takes about second to load which is why I added the loader screen when there is no data. 

I am still testing so let me see how it looks.

Reply:
I tested locally and it does take 2 seconds to load. However, I think the experience is fine. So I'll merge.